### PR TITLE
Changes for ros melodic compatibility.

### PR DESCRIPTION
**Pull request by ivandariojr**
Fix rosdep from libgazebo7-dev to libgazebo9-dev
Fix demo.py usage of addSolidPrimitive param wait -> use_service

### CODE REVIEW

File: fetch_gazebo_demo/scripts/demo.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def updateScene(self):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         wait = False)
+                                         use_service = False)
```
Comment (mikeferguson) [inline fetch_gazebo_demo/scripts/demo.py]: I think you actually want to just remove the "wait = False" -- the new service interface is quite speedy and very reliable (unlike the old "wait for sync").

Comment (mikeferguson) [inline fetch_gazebo_demo/scripts/demo.py:152]: I think you actually want to just remove the "wait = False" -- the new service interface is quite speedy and very reliable (unlike the old "wait for sync").

**Comment by moriarty** [comment]
Sorry for the delay, the first part of the change

```diff
- libgazebo7
+ libgazebo9
```
went in with #25

If you want to apply the suggestion from @mikeferguson's review I'll merge this.

**Comment by moriarty** [comment]
These changes went in via #32

**Pull request closed without merging**